.travis.yml

```diff
@@ -1,5 +1,4 @@
 language: node_js
 node_js:
-  - "4"
   - "6"
-  - "7"
+  - "8"
```

Comment:
@mantoni out of curiosity, would you bump a major for this?

Reply:
Adding 8 would be a minor. Removing 4 is a major. Running builds on uneven node is basically just being curious about the upcoming release and has no say in versioning. I tend to only run builds on LTS versions these days.